app/src/main/java/org/dhis2/utils/filters/PeriodFilterHolder.java

```diff
@@ -84,18 +84,8 @@ private void setListeners(FilterPeriodBinding periodLayout) {
         periodLayout.thisMonth.setOnCheckedChangeListener(this);
         periodLayout.lastMonth.setOnCheckedChangeListener(this);
         periodLayout.nextMonth.setOnCheckedChangeListener(this);
-        periodLayout.fromTo.setOnClickListener(view -> {
-            if (periodLayout.fromTo.isChecked()) {
-                updateSelection(R.id.fromTo);
-                FilterManager.getInstance().addPeriodRequest(FilterManager.PeriodRequest.FROM_TO);
-            }
-        });
-        periodLayout.other.setOnClickListener(view -> {
-            if (periodLayout.other.isChecked()) {
-                updateSelection(R.id.other);
-                FilterManager.getInstance().addPeriodRequest(FilterManager.PeriodRequest.OTHER);
-            }
-        });
+        periodLayout.fromTo.setOnCheckedChangeListener(this);
+        periodLayout.other.setOnCheckedChangeListener(this);
```

Comment:
This was using the onclicklistener to be able to select again these filters even if they are already selected.
Have you test to click them once they are checked?

Reply:
Ok, will have to change it. Didn't know that that was a possibility.

Reply:
Yep, it was discussed in this [issue](https://jira.dhis2.org/browse/ANDROAPP-2401)